cypress/integration/features/deals/all-deals/view-all-deals.feature

```diff
@@ -12,3 +12,13 @@ Feature: View All Deals
     Scenario: View partnered deals
         When I select Partnered Deals tab
         Then I can see Partnered Deals
+
+    @focus
+    Scenario: User can successfully change Metamask accounts
+        Given I'm not connected to a wallet
+        And I connect to the wallet with address "0x0000000000000000000000000000000000000000"
+        And I Wait for the modal with the message "Thank you for your patience while we initialize for a few moments..." to disappear
+        And I change the address to "0x0000000000000000000000000000000000000001"
+        And I Wait for the modal with the message "Thank you for your patience while we initialize for a few moments..." to disappear
+        Then The modal with the message "Thank you for your patience while we initialize for a few moments..." is hidden
+        And I'm connected to my wallet with address "0x0000000000000000000000000000000000000001"
```

Comment:
3 Notes

1. Metamask is not our app, so imo, we shouldn't have this as a Scenario, which relates to our app...
2. ... For this, I suggest to add either another gherkin tag (eg. "@thirdParty"), or use existing @regression
3. I would have written this differently, as the current way is too explicit about the UI:
```feature
Scenario: ...
  Given ...
  And I successfully connect to the wallet ...
  When my Wallet gets changed by a 3rd party 
  Then I'm connected to the new address
```

This way, we might be able to address 1. as well.
Optionally, hard-coding the address is fine for illustration purposes, but would have been better to use an more obvious example

Reply:
Also, as I understand it, was this bug fixed?
Or will it break currently?

Reply:
I did some more tests and it seems, even if the bug is fixed, the test is not quite correct. I shall modify it